pkg/cmd/util/clientcmd/factory.go

```diff
@@ -452,6 +468,25 @@ func (f *Factory) Clients() (*client.Client, *kclient.Client, error) {
 	return osClient, kClient, nil
 }
 
+// OriginSwaggerSchema returns a swagger API doc for an Origin schema under the /oapi prefix.
+func (f *Factory) OriginSwaggerSchema(client *kclient.RESTClient, version unversioned.GroupVersion) (*swagger.ApiDeclaration, error) {
+	if version.IsEmpty() {
+		return nil, fmt.Errorf("groupVersion cannot be empty")
+	}
+	path := "/swaggerapi/oapi/" + version.Version
```

Comment:
sort of surprised our policy allows this... is it installed outside the protected handler?